great_expectations/data_context/data_context/base_data_context.py

```diff
@@ -384,7 +387,7 @@ def add_store(self, store_name: str, store_config: dict) -> Optional[Store]:
             store (Store)
         """
 
-        self.config["stores"][store_name] = store_config
```

Comment:
Oooh good catch. Can we still access via ["stores"]? I think so because DataContextConfig is a subclass of DictDot. If not this would be a breaking change.